sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/src/main/java/com/azure/spring/sample/aad/controller/SampleController.java

```diff
@@ -41,7 +41,7 @@
      * @return Response with graph data
      */
     @GetMapping("call-graph-only")
-    @PreAuthorize("hasAuthority('SCOPE_ResourceAccessGraph.read')")
+    @PreAuthorize("hasAuthority('SCOPE_ResourceAccessGraph.Read')")
```

Comment:
Can we change this scope to Graph.Read?

Reply:
Oh, I think we could call these scopes Obo.Graph.Read and Obo.File.Read?

Reply:
Here I chose **Obo.Graph.Read** in the end.